lodash/index.d.ts

```diff
@@ -12701,7 +12701,7 @@ declare namespace _ {
          * @param value The value to check.
          * @returns Returns true if value is correctly classified, else false.
          */
-        isWeakMap<K, V>(value?: any): value is WeakMap<K, V>;
+        isWeakMap<K, V>(value?: any): value is WeakMapLike<K, V>;
```

Comment:
I do not think so. this change seems good to me, but we just need to remove the weakMap reference altogether.